gtfs/models/feed.py

```diff
@@ -13,8 +13,27 @@ def for_maas_operator(self, maas_operator: MaasOperator):
         )
 
 
-class Feed(TimestampedModel):
-    name = models.CharField(verbose_name=_("name"), max_length=255)
+class Feed(models.Model):
+    created_at = models.DateTimeField(verbose_name=_("created at"), auto_now_add=True)
+    name = models.CharField(
+        verbose_name=_("name"),
+        max_length=255,
+        blank=True,
+        help_text=_(
+            "If left empty, will be autopopulated with the feed info's publisher name when one is available."
+        ),
+    )
+    url_or_path = models.CharField(verbose_name=_("URL"), max_length=255)
```

Comment:
I guess the `verbose_name` is fine here (or do we want to add "or path"? :shrug:

Reply:
I think it is better because it should be a URL practically always, with the only exception being some local development stuff.